src/gql/sendPushNotification.gql.ts

```diff
@@ -0,0 +1,8 @@
+export const cqlSendPushNoticifation = `
```

Comment:
Please correct the typo. I suspect it should be gqlSendPushNoticifation :)

Reply:
Fixed in my latest commit